src/components/menu/LoadModule.js

```diff
@@ -187,6 +191,30 @@ class LoadModule extends Component {
         </div>
         )
 
+    case 'local-storage':
+      return (
+        <div className="col-4 nopadding">
```

Comment:
There's (currently) no way to navigate "subfolders" in local storage here, so we don't need to leave space for additional columns to the right. I think this outer `<div>` can be removed. 
![image](https://user-images.githubusercontent.com/13512036/126698558-bbe9ebb8-a578-49b4-baf1-3d0e46a5c839.png)

(compare to the "Core Modules" and "Submodules" lists which take up the full width)
![image](https://user-images.githubusercontent.com/13512036/126698641-3386d4b1-c86a-42db-a460-f729ac08f50c.png)